bundles/org.openhab.binding.insteon/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,444 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="insteon"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="network">
+		<label>Insteon Network</label>
+		<description>An Insteon PLM or hub that is used to communicate with the Insteon devices.</description>
+
+		<config-description>
+			<parameter name="port" type="text" required="true">
+				<label>Port</label>
+				<description>Configuration information that is used to connect to PLM or hub.</description>
+			</parameter>
+
+			<parameter name="devicePollInterval" type="integer" min="5000" max="3600000">
+				<label>Device Poll Interval</label>
```

Comment:
May use seconds here?

Reply:
Somebody might want to do finer grain tuning than at the seconds level. For example 2750 milliseconds.